### DISCUSSION TITLE
18. Built-in functions template.json.tpl como foi criado ?

### DISCUSSION THREAD

**Discussion by mazolatec** [chgasparoto/curso-aws-com-terraform#153] [Q&A]
Olá Cleber, muito obrigado por expor este conhecimento, gosto muito de tecnologias.
Gostaria de saber como vc criou arquivo template.json.tpl ??

**Comment by chgasparoto** [comment]
@mazolatec eu copiei de alguma ferramenta online que gera json com propriedades aleatórias mas eu não me lembro qual.
